packages/components/src/Battery/Battery.tsx

```diff
@@ -1,17 +1,24 @@
-/* eslint max-len: 0 */
 import React from 'react';
-import propTypes from 'prop-types';
-
 import classNames from 'classnames';
-
 import CriticalBattery from './CriticalBattery';
 import HighBattery from './HighBattery';
 import MediumBattery from './MediumBattery';
 import LowBattery from './LowBattery';
 import NullBattery from './NullBattery';
-
 import './battery.scss';
 
+/**
+ * Allowed values for severity which determines a variant of displayed Battery component.
+*/
+export type BatterySeverity = 1 | 2 | 3 | 4 | 'info' | 'low' | 'warn' | 'medium' | 'error' | 'high' | 'critical' | 'null';
```

Comment:
I think the `"null"` option is a bit weird. Is it referenced somewhere in the battery logic?

Reply:
@Hyperkid123 it's probably not needed, removed it. Thank you :slightly_smiling_face: